drape/vulkan/vulkan_layers.cpp

```diff
@@ -27,17 +27,9 @@ char const * const kDeviceExtensions[] =
   "VK_KHR_swapchain"
 };
 
-// DO NOT reorder. The order matters here.
 char const * const kValidationLayers[] =
 {
-  "VK_LAYER_GOOGLE_threading",
-  "VK_LAYER_LUNARG_device_limits",
-  "VK_LAYER_LUNARG_core_validation",
-  "VK_LAYER_LUNARG_image",
-  "VK_LAYER_LUNARG_object_tracker",
-  "VK_LAYER_LUNARG_parameter_validation",
-  "VK_LAYER_LUNARG_swapchain",
-  "VK_LAYER_GOOGLE_unique_objects",
+  "VK_LAYER_KHRONOS_validation",
```

Comment:
Why only this layer?

Reply:
This layer is the one and only now, other were removed several years ago